contents/docs/tutorials/cohorts.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Analyzing User Behavior with Cohorts
+sidebar: Docs
+showTitle: true
+---
+
+<br />
+
+<small class="note-block centered">_Estimated Reading Time: 7 minutes ☕☕_</small>
+
+<br />
+
+<span class="larger-image">
+
+![Cohorts banner image](../../images/tutorials/banners/cohorts.png)
+
+</span>
+
+<br />
+
+Each user uses your product slightly differently from the next, and some user groups might use your app in a completely different way from others.
+
+This happens for a variety of reasons, including software, user profiles, accesibility needs, culture, understanding of the product, individual goals, among many other variables.
+
+As such, an established technique for performing analytics is cohort analysis. When doing cohort analysis, users are split into different groups according to certain characteristics, and you can then derive metrics for each specific group (cohort), rather than only performing your analytics on the entire collection of users.
+
+This is extremely useful for comparing metrics across cohorts in order to determine areas for improvement, and maybe even product-market fit.
+
+Here are some examples:
+
+**Segmenting by browser**
+
+Allows you to determine if your UX is particularly poor on certain browsers. This can be indicated by a lower retention, bad conversion on your funnels, or a lack of certain events being captured. 
+
+**Segmenting by audience type**
```

Comment:
I'd clarify 'audience type' -> 'profile data', as you are specifically talking about user profile info here versus ie the geolocation of their computer that could be generated automatically